lib/common/nvpair.c

```diff
@@ -463,14 +479,15 @@ pcmk__cmp_nvpair_blocks(gconstpointer a, gconstpointer b, gpointer user_data)
     const gint a_is_higher = unpack_data->overwrite? 1 : -1;
     const gint b_is_higher = -a_is_higher;
 
-    if (a == NULL && b == NULL) {
-        return 0;
-    } else if (a == NULL) {
-        return b_is_higher;
+    // NULL values have lowest priority
```

Comment:
// no reason to turn that around for negative scores of the other that is != NULL